meson.build

```diff
@@ -360,6 +358,11 @@ endif
 # Hard-coded
 non_configurable_ops = [
     'FMiniIconsSupported',
+    'FORK_CREATES_CHILD',
+    'HAVE_MKSTEMP', # POSIX 2001; safer since 2008; no real need to check for this on a modern system.
```

Comment:
You can drop this comment; it's covered on Linux by `_DEFAULT_SOURCE` which we define.

Reply:
I have removed `HAVE_MKSTEMP` however kept the tidyup and still define `HAVE_SAFETY_MKSTEMP` - As far as I can tell if we're relying on _DEFAULT_SOURCE we're POSIX 2008 is safer (at least based on the `man`) down the line we may be able to drop the 'host does not have safe mkstemp' path.

I am happy to copy the code check from autotools if you would prefer that we properly check for all platforms :)